Dockerfile

```diff
@@ -25,15 +25,17 @@ RUN yarn build
 
 FROM golang:1.16.1-alpine3.13 as go-builder
 
-RUN apk add --no-cache gcc g++
+RUN apk add --no-cache gcc g++ git
 
 WORKDIR $GOPATH/src/github.com/credativ/plutono
 
 COPY go.mod go.sum build.go package.json ./
 COPY pkg pkg
 
-RUN go mod verify
-RUN go run build.go build
+RUN --mount=type=secret,id=GIT_AUTH_TOKEN \
+    git config --file $HOME/.gitconfig url.https://$(cat /run/secrets/GIT_AUTH_TOKEN)@github.com/.insteadOf https://github.com/ && \
+    go run build.go build && \
+    rm $HOME/.gitconfig
```

Comment:
Do we need this now once the repository is public?
What happened to go mod verify?

Reply:
Not needed anymore, reverted.